test/remote/sms_tests.js

```diff
@@ -38,3 +39,73 @@ describe('remote sms', () => {
     return TestServer.stop(server)
   })
 })
+
+describe('remote sms (mocked nexmo)', () => {
+  let server
+
+  before(() => {
+    config.sms.enabled = true
+    // POST /sms spends actual money unless the SMS provider is mocked
+    config.sms.useMock = true
+
+    return TestServer.start(config)
+      .then(result => {
+        server = result
+      })
+  })
+
+  it('POST /sms success', () => {
```

Comment:
No particular reason to add it besides to cover all endpoints. There is definitely a good amount of overlap between remote/local tests, but I have seen situations where one catches something the other doesn't. I think in this scenario it is ok not to add 👍